### PR TITLE
Adding filter clear message

### PR DESCRIPTION
**Pull request by Gerson2102**
Solves #94

### CODE REVIEW

File: src/network/protocol/messages/filterclear.zig [hunk 1]
```diff
@@ -0,0 +1,57 @@
+
```
Comment (tdelabro) [inline src/network/protocol/messages/filterclear.zig:1]: empty lane

File: src/network/protocol/messages/filterclear.zig [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+const std = @import("std");
+const native_endian = @import("builtin").target.cpu.arch.endian();
```
Comment (tdelabro) [inline src/network/protocol/messages/filterclear.zig:3]: you are not using this import

**Review by tdelabro — APPROVED** [review]

**Comment by tdelabro** [comment]
@Gerson2102 few fixes and resolve conflicts

**Comment by Gerson2102** [comment]
I think we are good now sir. Let me know @tdelabro

File: src/network/protocol/messages/filterclear.zig [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+const protocol = @import("../lib.zig");
+
+/// FilterClear represents the "filterclear" message
+///
+/// https://developer.bitcoin.org/reference/p2p_networking.html#filterclear
+pub const FilterClearMessage = struct {
+    // FilterClear message do not contain any payload, thus there is no field
+
+    pub inline fn name() *const [12]u8 {
+        return protocol.CommandNames.FILTERCLEAR ++ [_]u8{0} ** 5;
```
Comment (tdelabro) [inline src/network/protocol/messages/filterclear.zig:11]: `FILTERCLEAR` is already 11 char + final `\0`. No need to append anything to it in order to reach `*const [12]u8`

**Review by tdelabro — CHANGES_REQUESTED** [review]

**Review by tdelabro — APPROVED** [review]

**Pull request merged**